src/foam/core/XML.js

```diff
@@ -224,9 +239,24 @@ foam.CLASS({
           }
         },
         FObject: function(v, p) {
-          this.start('<' + this.propertyName(p) + '>');
-          this.output(p.toXML(v, this));
-          this.end('</' +  this.propertyName(p) + '>');
+          if ( v.xmlValue ) {
+            // if v.xmlValue exists then we have attributes
```

Comment:
Is this true? Aren't the use of attributes and a default xmlValue independent? Couldn't you have attributes with tags or a default xmlValue without attributes?

Reply:
I suppose you could have an xmlValue property without any attributes but can you think of the use case for that? You would be needlessly wrapping the true value in an extra class.

Reply:
<font size="+1">blah blah blah</font>, but I guess we're not going to be modelling HTML (actually, I suppose we do already model XML with U2), but we can leave it until we ever need it.